src/Core/Services/Implementations/BaseIdentityClientService.cs

```diff
@@ -52,12 +52,18 @@ public abstract class BaseIdentityClientService : IDisposable
         protected HttpClient IdentityClient { get; private set; }
         protected string AccessToken { get; private set; }
 
-        protected async Task SendAsync(HttpMethod method, string path, object requestModel = null)
+        protected Task SendAsync(HttpMethod method, string path) =>
+            SendAsync<object, object>(method, path);
+        
+        protected Task SendAsync<TRequest>(HttpMethod method, string path, TRequest body) =>
+            SendAsync<TRequest, object>(method, path, body);
```

Comment:
This overload seems a little weird, why's this different than line 61?

Reply:
My intention was to allow calls that don't require a response, thus not having to identity a `TResult`